### PR TITLE
[OSDOCS-11156]:What's new for OSD on GCP: Now support standard instance types for Master and Infra Nodes

### PR DESCRIPTION
**Pull request by mletalie**
Version(s):

4.16+
Issue:

https://issues.redhat.com/browse/OSDOCS-11156
Link to docs preview:

https://78418--ocpdocs-pr.netlify.app/openshift-dedicated/latest/osd_whats_new/osd-whats-new.html#osd-q3-2024_osd-whats-new
QE review:
- [ ] QE has approved this change.

**Reviewers: QE review N/A as this PR involves a release note for already QE review/approved doc ticket here: https://issues.redhat.com/browse/OSDOCS-10434
Also please note that formatting change to "Openshift Dedicated Regions added" release note was based on a suggestion here for an already merged PR https://github.com/openshift/openshift-docs/pull/78309: 
https://redhat-internal.slack.com/archives/D04RVNKPY14/p1719931809909369**

Additional information:

### CODE REVIEW

File: osd_whats_new/osd-whats-new.adoc [hunk 1]
```diff
@@ -17,15 +17,22 @@ With its foundation in Kubernetes, {product-title} is a complete {OCP} cluster p
 
 [id="osd-q3-2024_{context}"]
 === Q3 2024
+
+* **Expanded support for GCP standard instance types.** {product-title} on {GCP} now supports standard instance types for control plane and infrastructure nodes.
```
Comment (dfitzmau) [inline osd_whats_new/osd-whats-new.adoc:21]: I'm not seeing exactly the location where QE ack'ed this exact text. Would you have a link?

**Review by dfitzmau — APPROVED** [review]
/remove-label peer-review-in-progress

/label peer-review-done

File: osd_whats_new/osd-whats-new.adoc (hunk 1 above)
Comment (mletalie) [reply osd_whats_new/osd-whats-new.adoc:21]: Hello @dfitzmau, 
    Thanks for reviewing. Please see PR which includes a QE review/approval for this feature: https://github.com/openshift/openshift-docs/pull/75455 (specifically this comment which has the lgtm https://github.com/openshift/openshift-docs/pull/75455#issuecomment-2151720950) 
Release notes are a recap of a feature that has been approved/published. In this example the release note points out that we have expanded standard instance types for control plane and infrastructure nodes (previously we only supported standard instance types for worker nodes.)   See original feature ticket description below (in image) . 
image

**Pull request merged**